### PR TITLE
Information about fork in readme

### PR DESCRIPTION
**Pull request by mjarkk**
I've added some information about our fork to the README.  

Could someone review these changes and confirm they are correct?

### CODE REVIEW

**Review by glvr182 — APPROVED** [review]

**Comment by glvr182** [comment]
I would however just rebase this instead of merging to get a cleaner log

**Comment by mjarkk** [comment]
I'll wait for Skanehira's review and after that merge this in.

**Comment by skanehira** [comment]
LGTM :)

**Review by skanehira — APPROVED** [review]

**Pull request merged**
